src/main/java/mate/academy/spring/service/RoleService.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.spring.service;
+
+import mate.academy.spring.model.Role;
+
+public interface RoleService {
+    Role add(Role role);
+
+    Role getByName(Role.RoleName roleName);
```

Comment:
![image](https://user-images.githubusercontent.com/55763108/236467719-f7863b28-9e46-4c5e-b49d-70879cfeeb1c.png)
according to the requirements of the task, the input parameter must be of type String